portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +29,109 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadIntro(){
+    document.getElementById("play_button").style.visibility = "hidden"
+    //const info = "Hi I'm Cynthia. I'm a sophomore studying Computer Science and Statistics at Cornell University. I like photography and playing tennis and traveling. I was born and raised in New York and spend most of my time taking pictures, hanging out with my friends, or learning new things. I like to read and do my best to keep reading even when life gets hectic." ;
+    document.getElementById('introTitle').style.display = "block";
+    document.getElementById('introInfo').style.display = "block";
+    document.getElementById('profesh1').style.display = "block";
+    document.getElementById('profesh2').style.display = "block";
+    document.getElementById('profesh3').style.display = "block";
+        //document.getElementById('profesh5').style.display = "block";
+
+    var pics_len = document.getElementsByClassName("pics").length
+    var pics = document.getElementsByClassName("pics");
+    for(var i =0; i<pics_len; i++){
+        
+        (pics[i]).style.display = "none";
+        
+    }
+   
+    document.getElementById('profesh4').style.display = "none";
+    document.getElementById('next').style.display = "none";
+    document.getElementById('photo_intro').style.display = "none";
+    document.getElementById("prev").style.display="none";
+
+	//j=0;
+
+}
+
+
+
+function getPhotos_next(){
+    j++;
+    const elts = ["introTitle", "introInfo", "profesh1", "profesh2", "profesh3"];
+    const len = elts.length;
+    for(var i =0; i<len; i++){
+        document.getElementById(elts[i]).style.display = "none";
+    }
+    document.getElementById('photo_intro').style.display = "block";
+    
+    var pics_len = document.getElementsByClassName("pics").length
+    var pics = document.getElementsByClassName("pics");
+    for(var i =0; i<pics_len; i++){
+        if(i==j%6){
```

Comment:
It's not clear to me what this code is doing, so it might be worthwhile to add comments describing what i and j are (or even better, name the variables with more descriptive power).